README.md

```diff
@@ -24,42 +24,42 @@ git clone https://github.com/wso2-ballerina/module-amazonsqs.git
 
 ### Working with Amazon SQS Connector
 
-First, import the `wso2/amazonsqs` module into the Ballerina project.
+First, import the `ballerinax/aws.sqs` module into the Ballerina project.
 
 ```ballerina
-import wso2/amazonsqs;
+import ballerinax/aws.sqs;
 ```
 
 In order for you to use the Amazon SQS Connector, first you need to create an Amazon SQS Client.
 
-Ballerina provides a [config module](https://ballerina.io/learn/api-docs/ballerina/config.html) to obtain parameters from the configuration file. Specify the configuration object and create the client as follows.
+Ballerina provides a [config module](https://ballerina.io/v1-1/learn/by-example/config-api.html) to obtain parameters from the configuration file. Specify the configuration object and create the client as follows.
 
 ```ballerina
-amazonsqs:Configuration configuration = {
+aws.sqs:Configuration configuration = {
```

Comment:
> "Module names can contain alphanumeric characters including dots (.). Dots in a module name has no meaning other than the last segment after the final dot being used as a default alias within your source code."
```suggestion
sqs:Configuration configuration = {
```

Reply:
As per the docs, we can just use the final part for the module name. So shall we change it accordingly?

Reply:
Changed in all locations.